src/client/components/ProductDetails/ProductDetails.component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import NumberInput from '../NumberInput/NumberInput.component'
+import ButtonComponent from '../Button/Button.component'
+import './ProductDetails.style.css'
+import Heart from './Heart';
+
+
+export const ProductDetails = (
+    {
+        imgSource,
+        ProductName,
+        RemainingUnit,
+        Price,
+        productColor,
+        productSize,
+        onClick,
+        isFavorite,
+        imageAlt,
+    }
+) => {
+    const checkFavorite = () => {
+        console.log('checked') // This function need to be change database for add or remove from favorite.
+    }
+
+    return <div id="product-details">
+        <div className='product-image'>
+            <img src={imgSource} alt={imageAlt} />
+        </div>
+        <div className='details-column'>
+            <div>
+                <div className='product-name' >
+                    {ProductName}
+                    <button type='button' onClick={checkFavorite}>
+                        <div className='heart'> {isFavorite ? <Heart height='30'/> : <Heart height='30' fill='#8E0EF2' strokeWidth='0'/>} </div>
+                    </button>
+                </div>
+                <small>({RemainingUnit} units left)</small>
+            </div>
+            <div className='price'>
+                <b>{Price} DKK</b>
+            </div>
+            <div className='input-row'>
+                <div className='select-item'>
+                    <select>
+                        <option>Color</option>
+                        <option>{productColor}</option>
+                    </select>
+                    <div className="arrow-right" />
```

Comment:
<div className="arrow-right" /> is outside of <select>, so it doesnot work on click.

Reply:
Hi Zahra. I fix this part with css.